addons/zookeeper/templates/cmpd.yaml

```diff
@@ -63,15 +52,8 @@ spec:
           - -c
           - |
             . "$ZOOBINDIR"/zkEnv.sh > /dev/null
-
-            isSSL="false"
-            clientPortAddress="localhost"
-            clientPort=2181
-            GREP=grep
-            STAT=`"$JAVA" "-Dzookeeper.log.dir=${ZOO_LOG_DIR}" "-Dzookeeper.log.file=${ZOO_LOG_FILE}" \
-            -cp "$CLASSPATH" $CLIENT_JVMFLAGS $JVMFLAGS org.apache.zookeeper.client.FourLetterWordMain \
-            $clientPortAddress $clientPort srvr $isSSL 2> /dev/null    \
-            | $GREP Mode`
+            STAT=$(java -cp "$CLASSPATH" $CLIENT_JVMFLAGS $JVMFLAGS org.apache.zookeeper.client.FourLetterWordMain \
```

Comment:
Please synchronize with the latest version and API from the main branch. In the new API, `builtinHandler: custom` is no longer available. Please use the kbagent approach for integration.